src/main/java/ie/ait/bteam/drcare/DrCareApplication.java

```diff
@@ -3,18 +3,21 @@
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.boot.builder.SpringApplicationBuilder;
+import org.springframework.boot.web.servlet.ServletComponentScan;
 import org.springframework.boot.web.servlet.support.SpringBootServletInitializer;
 
 @SpringBootApplication
-public class DrCareApplication extends SpringBootServletInitializer {
```

Comment:
please put extends back to expose the controllers to outside spring context.

Reply:
Done...